index.html

```diff
@@ -473,7 +473,17 @@ <h3>
             "<dfn data-dfn-for="text-direction">auto</dfn>" (default)
           </dt>
           <dd>
-            No explicit directionality.
+            <p>
+              Direction determined from content using the first-strong part of
```

Comment:
Replace "the first strong part" with "Rule P2". (Elsewhere, we link to those rules like this: `<a data-cite="bidi#P2">Rule P2</a>` -- not sure if there is a more Respeccy way to link that.)

Note: Even though it doesn't explicitly say this, I consider this to be non-normative text (since it is just giving a human-readable summary of what this value means). The "when displaying" algorithm below gives the actual normative statement. So it isn't necessary to have this be entirely precise, but I think it helps to link to the exact part of the algorithm.